TrafficSimulator/Assets/RoadGenerator/Core/Runtime/Objects/BezierPath.cs

```diff
@@ -5,6 +5,24 @@
 
 namespace RoadGenerator
 {
+	struct SegmentIntersection
```

Comment:
All structs have wrong coding standrad. Since the whole file have wrong coding standard it's fine to do keep it like this. Maybe the struct that is used outside of this file should change